jupyterhub/assignments/intake-dashboard/whats_happening_where.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\", category=FutureWarning)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext sql\n",
+    "%config SqlMagic.feedback = False\n",
+    "%config SqlMagic.displaylimit = 0\n",
+    "%config SqlMagic.displaycon = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "\n",
+    "start = datetime.datetime.now() - datetime.timedelta(days=90)\n",
+    "end = datetime.datetime.now()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql report_locations <<\n",
+    "\n",
+    "SELECT location_name, COUNT(*) count\n",
+    "FROM cts_forms_report\n",
+    "WHERE location_name notnull\n",
+    "GROUP BY location_name\n",
+    "HAVING COUNT(*) > 1\n",
+    "ORDER BY count desc"
```

Comment:
Add `LIMIT 10` after this order by to pull only ten records back into python

(this will save a lot of RAM)